docs/debugging-resources/prow.md

```diff
@@ -26,16 +26,19 @@ Configuration files are found in this folder: https://github.com/kubevirt/projec
 
 **Note: some of these need to be a symlink to the real files, please follow these and edit in target location.**
 
-| Type         | Holds configuration for                    |
-|--------------|--------------------------------------------|
-| orgs.yaml    | KubeVirt GitHub organization               |
-| config.yaml  | Prow components, prow job presets          |
-| plugins.yaml | Prow plugins                               |
-| labels.yaml  | GitHub labels for pull requests and issues |
-| mirror.yaml  | image mirror                               |
-| jobs         | job configurations per repository          |
-
+| Type         | Holds configuration for                |
+|--------------|----------------------------------------|
+| config.yaml  | Prow components, prow job presets      |
+| plugins.yaml | Prow plugins                           |
+| mirror.yaml  | image mirror                           |
+| jobs         | Prow job configurations per repository |
+| labels.yaml  | GitHub labels[^1]                      |
+| orgs.yaml    | GitHub organization settings[^2]       |
 
+[^1]: GitHub labels for pull requests and issues are updated periodically by cron job [label-sync-kubevirt] and [label-sync-nmstate]
```

Comment:
```suggestion
[^1]: GitHub labels for pull requests and issues are updated periodically by the cron jobs [label-sync-kubevirt] and [label-sync-nmstate].
```
markdown doesn't respect the newline when built so we need a fullstop to distinguish sentences.